public/src/main/java/com/revenuecat/purchases/models/PurchaseDetails.kt

```diff
@@ -12,7 +11,7 @@ import org.json.JSONObject
 @TypeParceler<JSONObject, JSONObjectParceler>()
 // TODO: docs
 data class PurchaseDetails(
-    val orderId: String,
+    val orderId: String?,
```

Comment:
We only have `orderId` in the `Purchase`. `PurchaseHistoryRecord` doesn't provide an `orderId` and Amazon doesn't have that concept either. I think if we document it we should be fine.